docker/README.md

```diff
@@ -73,9 +73,11 @@ If you are using Docker for Mac or Docker for Windows (or the older Docker Toolb
 
 While running the `oldev` container, gunicorn is configured to auto-reload modified files. To see the effects of your changes in the running container, the following apply:
 
-- Editing python files or web templates => simply save the file, gunicorn will auto-reload it.
-- Working on frontend css or js => you must run `docker-compose exec web make css js`. This will re-generate the assets in the persistent `ol-build` volume mount, so the latest changes will be available between stopping / starting and removing `web` containers. Note, if you want to view the generated output you will need to attach to the container (`docker-compose exec web bash`) to examine the files in the volume, not in your local dir.
-- Adding or changing core dependencies => you will most likely need to rebuild both `olbase` and `oldev` images. This shouldn't happen too frequently. If you are making this sort of change, you will know exactly what you are doing ;)
+- **Editing python files or web templates?** Simply save the file; gunicorn will auto-reload it.
+- **Editing frontend css or js?** Run `docker-compose exec web npm run-script build-assets`. This will re-generate the assets in the persistent `ol-build` volume mount (so the latest changes will be available between stopping / starting  `web` containers). Note, if you want to view the generated output you will need to attach to the container (`docker-compose exec web bash`) to examine the files in the volume, not in your local dir.
```

Comment:
Is this correct? `docker-compose exec web npm run-script build-assets`  I thought the Make tasks handled the building of assets (they always have in the past, and I wasn't aware of a change in the process). This looks like it is adding another path. Do they do the same thing? If they do, we should only have one way to trigger them.

Reply:
I thought I saw this in the docs somewhere, but can't find it anymore. @jdlrobson what's the recommended command, `make js css` or `npm run-script build-assets`?